package org.kryptonmc.krypton.auth.exceptions

/**
 * Thrown when an error occurs attempting to authenticate a user
 */
class AuthenticationException : RuntimeException()
